packages/happy-dom/src/xml-http-request/XMLHttpRequest.ts

```diff
@@ -146,6 +146,15 @@ export default class XMLHttpRequest extends XMLHttpRequestEventTarget {
 		return this._state.statusText;
 	}
 
+	/**
+	 * Returns the response.
+	 *
+	 * @returns Response.
+	 */
+	public get response(): ArrayBuffer | Blob | IDocument | object | string {
+		return this._state.response;
+	}
+
```

Comment:
as you see, it's easy to add test unit for #788 

```suggestion
	/**
	 * Returns the response.
	 *
	 * @returns Response.
	 */
	public get response(): ArrayBuffer | Blob | IDocument | object | string {
		return this._state.response;
	}

```